advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        saveDir = pwd + '/data/'
</pre><p>I would probably put plotting in a function. Or is there a reason it needs to be explicit?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>